### DISCUSSION TITLE
Release strategy

### DISCUSSION THREAD

**Discussion by BastiLoe** [OpenRadioss/OpenRadioss#836] [Q&A]
Hello all,

is there any plan to do official releases of OpenRadioss or will it stay with nightly builds? Will there be some correlation to future Altair Radioss releases?

**Comment by PaulAltair** [comment]
Thank you for your interest and question, there is no plan to issue official 'releases' of OpenRadioss currently, and no plan to have specific 'Altair vX.X' equivalent builds, however the same fundamental source code is used to build both OpenRadioss and Altair Radioss, as such, future Altair Radioss releases will incorporate features and bug fixes previously added here but may also have some features rolled back depending on QA etc. OpenRadioss is in effect the nightly Radioss core feature development build and Altair Radioss releases are the commercial forks of that with some additional features (e.g. ability to use encrypted material data) and documentation. In terms of core code feature set, OpenRadioss will always be closest to (and slightly 'ahead' of) the latest Altair Radioss release.
